src/graph/topo.h

```diff
@@ -94,8 +94,7 @@ struct ncclTopoLink {
   float bw;
   struct ncclTopoNode* remNode;
 };
-#define NCCL_TOPO_MAX_LINKS 64	//Changed the value from 32 to 64 for CPX mode
-
+#define NCCL_TOPO_MAX_LINKS 128
```

Comment:
RCCL expanded this from 32 to 64 for CPX mode, but NCCL expands all the way to 128, so we've taken the larger value.